packages/core/test/util/normalizeConfig_test.js

```diff
@@ -785,6 +786,29 @@ describe('util.normalizeConfig', () => {
       const config = normalizeConfig();
       expect(config.tracing.ignoreEndpointsDisableSuppression).to.deep.equal(true);
     });
+
+    it('should return false when INSTANA_TRACING_DISABLE_EOL_EVENTS is not set', () => {
+      const config = normalizeConfig();
+      expect(config.tracing.disableEOLEvents).to.deep.equal(false);
```

Comment:
Since this is primitive value checking, we don't have to use `deep.equal`.
```suggestion
      expect(config.tracing.disableEOLEvents).to.equal(false);
```